src/coreclr/inc/corinfo.h

```diff
@@ -387,18 +387,18 @@ enum CorInfoHelpFunc
     CORINFO_HELP_ULMOD,
     CORINFO_HELP_LNG2DBL,               // Convert a signed int64 to a double
     CORINFO_HELP_ULNG2DBL,              // Convert a unsigned int64 to a double
-    CORINFO_HELP_DBL2INT,
+    CORINFO_HELP_DBL2INT,               // unused
     CORINFO_HELP_DBL2INT_OVF,
     CORINFO_HELP_DBL2LNG,
     CORINFO_HELP_DBL2LNG_OVF,
-    CORINFO_HELP_DBL2UINT,
+    CORINFO_HELP_DBL2UINT,              // unused
```

Comment:
I expect that #97529 is going to add dependency on this helper back.

Reply:
I think it makes more sense for them to remove the comment if this will be merged first and they'll end up needing the helper again.

Reply:
My preference would be to leave all changes related to floating point to integer conversions to the Intel PRs.